report/chapter1.tex

```diff
@@ -287,3 +287,34 @@ \subsubsection{"fork server"}
   \item{} Exécution du programme par \lstinline{afl-fuzz} \url{https://github.com/mirrorer/afl/blob/master/afl-fuzz.c#L2361-L2382};
   \item{} Et dans \url{https://github.com/mirrorer/afl/blob/master/afl-as.h}, avec les fonctions \lstinline{__afl_forkserver}, \lstinline{__afl_fork_resume}, \lstinline{__afl_fork_wait_loop} et la variable \lstinline{__afl_fork_pid}, le code injecté dans le programme cible.
 \end{itemize}
+
+\subsection{Le ``persistent mode''}\label{persistent-mode}
+
+Une autre optimisation implémentée dans AFL est le ``persistent mode''.
+Celui-ci est encore expérimental et permet de bénéficier des avantages de performances du fuzzing ``in-process'' et de la robustesse du fuzzing utilisant le ``fork server''.
```

Comment:
Je crois pas qu'on ait définit ce "in-process" autre part dans le rapport ?

Reply:
Non, je peux rajouter entre parenthèse : (c'est à dire que l'on ne recréer pas un processus pour chaque entrée et que l'on fuzz directement en mémoire)

Référence : [google blog](https://security.googleblog.com/2016/08/guided-in-process-fuzzing-of-chrome.html)

Reply:
Ouais ce serait nickel